src/libsync/logger.cpp

```diff
@@ -141,6 +141,10 @@ void Logger::doLog(QtMsgType type, const QMessageLogContext &ctx, const QString
         msgW.append(L"\n");
         OutputDebugString(msgW.c_str());
     }
+#endif
+#if defined(NEXTCLOUD_DEV)
+    QTextStream cout(stdout, QIODevice::WriteOnly);
+    cout << msg << endl;
```

Comment:
missed this during my initial review
I am not sure this is what you want
I often find myself needing to enable debug and immediate flush during dev work (even with release builds) and use the command line option `--logfile -` to get console output instead of modifying the internal code to do it

Reply:
I use this
![image](https://github.com/nextcloud/desktop/assets/168967/e1bab5ba-8129-40a7-8c63-35ccf85ea8e6)
that would not be needed on windows as there is a way to send logging output directly to the debugger

Reply:
so currently on windows if `NEXTCLOUD_DEV` is defined, I suggest that the output is effectively sent to the debugger
on other platforms, I still think using a command line option may be more sensible
the code you are adding is probably going to be pretty slow by building an instance of `QTextStream` for every line of output
@camilasan  what would prevent you from using `--logfile -` command line option ?